<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Giris_yap</class>
 <widget class="QWidget" name="Giris_yap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Ekranı</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 170, 165);</string>
  </property>
  <widget class="QLabel" name="kullanici_adi_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı :</string>
   </property>
  </widget>
  <widget class="QLabel" name="sifre_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Şifre :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kullanici_adi_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sifre_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="giris_yap_buton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_ol_buton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
  <widget class="QLabel" name="durum_giris_ekran_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
